docs/requirements.txt

```diff
@@ -2,7 +2,7 @@ sphinx
 pydata-sphinx-theme>=0.4.2
 sphinx_book_theme
 myst-parser
-myst-nb
+myst-nb==0.12.3
```

Comment:
```suggestion
myst-nb>=0.12.3
```

Reply:
This change should help us not get stuck on this particular version as myst-nb evolves.

Reply:
I've wondered why so many projects use `>=` dependency versioning; this makes perfect sense.